change/@fluentui-tokens-a1396b99-e2b7-4ab3-8c89-7d58c5b4ba8b.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentui-web-components-v3 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33393&commitId=9a45d5fedfbcb102d9fe6a6cee18d583fa17653b&env=prod&clientType=FLUENTUI)


  <details><summary><b>Accordion</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Accordion.Marker Position End Dark Mode.chromium.png|2302| Changed|
|Accordion.Accordion With Custom Icons Dark Mode.chromium.png|1109| Changed|
|Accordion.Size Dark Mode.chromium.png|9027| Changed|
|Accordion.Disabled Dark Mode.chromium.png|2617| Changed|

</details>


  <details><summary><b>Avatar</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar.Active Appearance Dark Mode.chromium.png|23| Changed|
|Avatar.Color Dark Mode.chromium.png|614| Changed|
|Avatar.Active Dark Mode.chromium.png|13| Changed|
|Avatar.Colorful Dark Mode.chromium.png|270| Changed|

</details>


  <details><summary><b>Badge</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Badge.Size Dark Mode.chromium.png|89| Changed|
|Badge.Shape Dark Mode.chromium.png|17| Changed|
|Badge.Color Dark Mode.chromium.png|2459| Changed|
|Badge.Appearance Dark Mode.chromium.png|472| Changed|

</details>


  <details><summary><b>Button</b> 15 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button.Outline Disabled Dark Mode.chromium.png|55| Changed|
|Button.Outline Dark Mode.chromium.png|224| Changed|
|Button.Medium With Icon Dark Mode.chromium.png|746| Changed|
|Button.Large With Icon Dark Mode.chromium.png|941| Changed|
|Button.Primary Dark Mode.chromium.png|171| Changed|
|Button.Outline Disabled Focusable Dark Mode.chromium.png|55| Changed|
|Button.Primary Disabled Dark Mode.chromium.png|74| Changed|
|Button.Primary Disabled Focusable Dark Mode.chromium.png|74| Changed|
|Button.Small With Icon Dark Mode.chromium.png|332| Changed|
|Button.Transparent Disabled Focusable Dark Mode.chromium.png|77| Changed|
|Button.Subtle Dark Mode.chromium.png|162| Changed|
|Button.Subtle Disabled Focusable Dark Mode.chromium.png|50| Changed|
|Button.Transparent Dark Mode.chromium.png|256| Changed|
|Button.Subtle Disabled Dark Mode.chromium.png|50| Changed|
|Button.Transparent Disabled Dark Mode.chromium.png|77| Changed|

</details>


  <details><summary><b>Divider</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Divider.Appearance Default Dark Mode.chromium.png|101| Changed|
|Divider.Appearance Strong Dark Mode.chromium.png|101| Changed|
|Divider.Appearance Brand Dark Mode.chromium.png|44| Changed|
|Divider.Appearance Subtle Dark Mode.chromium.png|68| Changed|

</details>


  <details><summary><b>Label</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Label.Default Dark Mode.chromium.png|144| Changed|
|Label.Disabled Dark Mode.chromium.png|46| Changed|
|Label.Required Dark Mode.chromium.png|166| Changed|

</details>


  <details><summary><b>MenuList</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|MenuList.Checkbox Dark Mode.chromium.png|269| Changed|
|MenuList.Default Dark Mode.chromium.png|117| Changed|
|MenuList.Radio Dark Mode.chromium.png|269| Changed|
|MenuList.With Submenu Dark Mode.chromium.png|269| Changed|

</details>


  <details><summary><b>ProgressBar</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ProgressBar.Default Dark Mode.chromium.png|1789| Changed|

</details>


  <details><summary><b>Tabs</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tabs.Size Large Vertical Dark Mode.chromium.png|1131| Changed|
|Tabs.Default Dark Mode.chromium.png|1083| Changed|
|Tabs.Size Medium Dark Mode.chromium.png|1083| Changed|
|Tabs.Disabled Dark Mode.chromium.png|312| Changed|
|Tabs.Size Large Dark Mode.chromium.png|1387| Changed|
|Tabs.Size Small Dark Mode.chromium.png|1075| Changed|
|Tabs.Size Medium Vertical Dark Mode.chromium.png|914| Changed|
|Tabs.Size Small Vertical Dark Mode.chromium.png|914| Changed|

</details>


  <details><summary><b>TextInput</b> 10 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TextInput.Appearance Filled Darker Dark Mode.chromium.png|377| Changed|
|TextInput.Appearance Filled Lighter Dark Mode.chromium.png|374| Changed|
|TextInput.Content Start Dark Mode.chromium.png|227| Changed|
|TextInput.Appearance Underline Dark Mode.chromium.png|355| Changed|
|TextInput.Content End Dark Mode.chromium.png|216| Changed|
|TextInput.Content Start End Dark Mode.chromium.png|532| Changed|
|TextInput.Appearance Outline Dark Mode.chromium.png|446| Changed|
|TextInput.Required Dark Mode.chromium.png|275| Changed|
|TextInput.Disabled Dark Mode.chromium.png|300| Changed|
|TextInput.Placeholder Dark Mode.chromium.png|662| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>